numpy/core/code_generators/numpy_api.py

```diff
@@ -370,6 +370,11 @@ def get_annotations():
     'PyDataMem_SetHandler':                 (304, MinVersion("1.22")),
     'PyDataMem_GetHandler':                 (305, MinVersion("1.22")),
     # End 1.22 API
+    'NpyDatetime_ConvertDatetime64ToDatetimeStruct': (307, MinVersion("1.25")),
+    'NpyDatetime_ConvertDatetimeStructToDatetime64': (308, MinVersion("1.25")),
+    'NpyDatetime_ConvertPyDateTimeToDatetimeStruct': (309, MinVersion("1.25")),
+    'NpyDatetime_GetDatetimeISO8601StrLen':          (310, MinVersion("1.25")),
+    'NpyDatetime_MakeISO8601Datetime':               (311, MinVersion("1.25")),
```

Comment:
Added the min-version and hopefully resolved a bit of a merge/rebase mess (the sorting submodule was updated in the diff).